pkg/server/proto/server.proto

```diff
@@ -137,6 +137,10 @@ service Waypoint {
   // GetDeployment returns a deployment
   rpc GetDeployment(GetDeploymentRequest) returns (Deployment) {}
 
+  // GetLatestDeployment returns the most recent successfully completed build
```

Comment:
```suggestion
  // GetLatestDeployment returns the most recent successfully completed Deployment
```

You will want to make sure to re-gen the protos here so the doc string is accurate in the generated method!